### PR TITLE
Update lib-helm and changed security context for deckhouse

### PR DESCRIPTION
**Pull request by duckhawk**
## Description

Changed security context for deckhouse pod, and removed bashible selinux policies steps

## Why do we need it, and what problem does it solve?

We did it to unify our workarounds for selinux support

## What is the expected result?

Working deckhouse pod with enabled selinux policies

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse
type: chore
summary: Changed security context for deckhouse pod, and removed bashible selinux policies steps.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/vsphere'...

File: candi/bashible/bundles/centos/node-group/033_install_mandatory_selinux_policies.sh.tpl [hunk 1]
```diff
@@ -12,64 +12,9 @@
...
-allow load_policy_t var_lock_t:file write;
-
-#============= setfiles_t ==============
-allow setfiles_t var_lib_t:file read;
-EOF
-
-{{- if eq .cri "Containerd" }}
 bb-event-on 'selinux_cilium_policy_changed' '_on_selinux_cilium_policy_changed'
 _on_selinux_cilium_policy_changed() {
   checkmodule -M -m -o /var/lib/bashible/policies/cilium.mod /var/lib/bashible/policies/cilium.te
```
Comment (RomanenkoDenys) [inline candi/bashible/bundles/centos/node-group/033_install_mandatory_selinux_policies.sh.tpl:20]: This file should be removed completely - we do not need any of selinux security policies. Security policies should be in components deloyments

Comment (duckhawk) [reply candi/bashible/bundles/centos/node-group/033_install_mandatory_selinux_policies.sh.tpl:20]: Security context is already set in cillium policies. I'll remove this file, it must be fine

**Comment by duckhawk** [comment]
> some questions

AFAIS fixed

**Comment by z9r5** [comment]
/e2e/abort openstack;Standard;containerd;1.23 5400733629 5400733629-1-con-1-23 /sys/deckhouse-oss/install:pr5028 debian@45.12.230.159

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Review by yalosev — DISMISSED** [review]

**Review by name212 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/openstack'...

**Review by name212 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by z9r5** [comment]
/e2e/abort static;Static;containerd;1.23 5456364104 5456364104-1-con-1-23 /sys/deckhouse-oss/install:pr5028 astra@192.168.199.213

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Review by yalosev — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/static'...

**Pull request merged**